substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/methodhandles/Target_java_lang_invoke_MethodHandles_Lookup.java

```diff
@@ -39,6 +41,11 @@
 
 @TargetClass(value = MethodHandles.class, innerClass = "Lookup")
 final class Target_java_lang_invoke_MethodHandles_Lookup {
+    // Checkstyle: stop
+    @Delete //
+    static ConcurrentHashMap<Target_java_lang_invoke_MemberName, MethodHandle> LOOKASIDE_TABLE;
+    // Checkstyle: resume
+
```

Comment:
This seems to be accidentally brought in from https://github.com/oracle/graal/pull/8122. Could you please elaborate on why it's needed?

Reply:
Same here, this is a bug that appeared during testing the metadata repository with converted metadata (from `*-config.json` to `reachability-metadata.json`). The likely cause is that `"type"` in `reachability-metadata.json` brings in the whole class reflection metadata for registered types instead of the more targeted registrations from the legacy files. In this case the bug was already discovered and fixed in oracle/graal#8122, so I included the fix in the backport.